go.mod

```diff
@@ -84,6 +85,7 @@ require (
 	k8s.io/kops v0.0.0-20190222135932-278e6606534e
 	k8s.io/kubelet v0.0.0-20190313123811-3556bcde9670
 	k8s.io/kubernetes v1.12.6
+	sigs.k8s.io/kustomize v2.0.3+incompatible // indirect
```

Comment:
I guess we don't need this anymore. Did you run `go mod tidy` ?

Reply:
No, I didn't know about it, thanks :+1: